website_anchor_smooth_scroll/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingeniería S.L. - Jairo Llopis
```

Comment:
@pedrobaeza is this line needed I don't now if it could be delete or not, or if we need to add all contributors.

Reply:
The idea is to collect here all the contributors, and when touched, on the respective files.